private/buf/cmd/buf/buf_test.go

```diff
@@ -2557,6 +3049,218 @@ func TestLintWithPaths(t *testing.T) {
 	)
 }
 
+func TestLintWithPlugins(t *testing.T) {
+	t.Parallel()
+	// defaults only, comment ignores on.
+	testRunStdout(
+		t,
+		nil,
+		bufctl.ExitCodeFileAnnotation,
+		filepath.FromSlash(`
+testdata/check_plugins/current/proto/api/v1/service.proto:11:1:Service name "api.v1.FooServiceMock" has banned suffix "Mock". (buf-plugin-suffix)
```

Comment:
Same thing here - I know we already have a bunch of these already though, and don't have any good options. Perhaps something like `--error-format=json` and then parse the JSON and make all the messages empty before comparison. That might be a lot of work, and we just want to get these across.

Reply:
Yeah... so @oliversun9 and I had a chat this morning about how to refactor some of these tests, because maintenance is definitely a concern. We like the idea of parsing the `json` output for the `ls-{lint, breaking}-rules` tests, but these lint and breaking one, we couldn't really think of a much nicer solution, since you need to structure the expected output somehow. We are proposing keeping these as-is for now unless we think of something really nice for them.